responses/es/01_update-index.md

```diff
@@ -0,0 +1,20 @@
+## Paso 2: Personaliza tu página inicial
+
+¡Buen trabajo! Ahora que GitHub Pages está activado, puedes ver tu sitio web aquí: {{ deploymentUrl }}
+
+Puedes personalizar tu página inicial añadiendo contenido bien a un archivo `index`  o al archivo `README.md`. GitHub Pages buscará primero un archivo `index`. Si encuentra un archivo `index`, GitHub Pages utiliza el contenido en ese archivo para crear la página inicial. Si no encuentra un archivo `index`, utiliza `REAMDE.md` para crear la página inicial.
+
+Tu repositorio tiene un archivo `index.md`, así que podemos actualizarlo para que incluya tu contenido personalizado.
+
+### :keyboard: Actividad: Crea tu página inicial
+
+1. Navega a la pestaña **Code** de este repositorio y accede al archivo `index.md`, o haz clic en [este enlace]({{ repoUrl }}/blob/master/index.md)
+2. En la esquina superior derecha, haz clic en el icono a la derecha del archivo para editar el archivo `index.md` 
+3. Escribe el contenido que quieres que aparezca en tu página principal
+4. Haz scroll hasta la parte inferior, escribe un mensaje de commit y haz clic en **Create a new branch for this commit and start a pull request**
+5. Abre una petición de extracción (pull request)
```

Comment:
🍺 Ok so here we are translating `Pull Request`. My personal preference would be to be consistent when translating feature names.

Reply:
Agree. However, again in personal experience with students, I tend to use original English names when talking commands, translated words when I'm using them in an explanation. I find students get confused when referring to a command by its translated name. My attempt here was to introduce the command through the explanation